<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SNILOAU" failures="1" tests="6" name="Re-testing" time="0.088" errors="0" timestamp="2024-08-27T10:09:56 BDT">
  <testcase classname="org.nadim.testscripts.TC009_TestNGDependentTestMethod" name="testMethod2" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [6] but found [7]">
      <![CDATA[java.lang.AssertionError: expected [6] but found [7]
at org.nadim.testscripts.TC009_TestNGDependentTestMethod.testMethod2(TC009_TestNGDependentTestMethod.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase classname="org.nadim.testscripts.TC009_TestNGDependentTestMethod" name="testMethod1" time="0.004"/>
  <testcase classname="org.nadim.testscripts.TC009_TestNGDependentTestMethod" name="testMethod3" time="0.005"/>
  <testcase classname="org.nadim.testscripts.TC001_SumTwoNumbers" name="testMethod1" time="0.003"/>
  <testcase classname="org.nadim.testscripts.TC001_SumTwoNumbers" name="testMethod2" time="0.003"/>
  <testcase classname="org.nadim.testscripts.TC001_SumTwoNumbers" name="testMethod3" time="0.005"/>
</testsuite> <!-- Re-testing -->
